kolibri/plugins/management/assets/src/device_management/views/manage-permissions-page/user-full-name-cell.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+
+  <span>
+
+    <span class="icon" ref="icon">
+      <mat-svg
+        v-if="permissionType==='SUPERUSER'"
+        category="toggle"
+        name="star"
+        class="superuser"
+      />
+      <mat-svg
+        v-if="permissionType==='SOME_PERMISSIONS'"
+        category="toggle"
+        name="star"
+        class="some-permissions"
+      />
+      <mat-svg
+        v-if="permissionType==='NO_PERMISSIONS'"
+        category="social"
```

Comment:
How do you go from a person to a star? haha Why not use `star_border` instead or just a gray `star`? Makes more sense.

Reply:
Actually why not avoid stars since we already use stars in Learn to denote completion?

Reply:
Different colored keys are a better analogy? https://material.io/icons/#ic_vpn_key

Reply:
User icon maps to our existing user icon, and conveys nothing out of the ordinary. My primary concern with the star_border was that it looked too much like the other stars. I wanted a way to visually separate those who have no permissions from those who have 1+ or superuser.

Reply:
But the user icon indicates a user type where as a star indicates permissions. If it's just filler, than why not leave out icons for users without permissions?

Reply:
We considered keys and locks early on but they're not universally understood in association with permissions

Reply:
> if we leave out icons for users without permissions, then the column becomes misaligned

The icons should be in their own column, not part of the `name` column.